### PR TITLE
DNS/Dnscrypt-proxy remove discontinued feeds

### PR DESCRIPTION
**Pull request by L1ghtn1ng**
This PR removes feeds that are discontinued or are not updated any more, 1 need feed addition and a fix for a feed

### CODE REVIEW

**Comment by L1ghtn1ng** [comment]
@mimugmail Can you have a look at this please to see if I need to change anything?

File: dns/dnscrypt-proxy/Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 PLUGIN_NAME=		dnscrypt-proxy
-PLUGIN_VERSION=		1.7
+PLUGIN_VERSION=		1.8
 PLUGIN_REVISION=	1
```
Comment (fichtner) [inline dns/dnscrypt-proxy/Makefile:3]: if we bump version we remove revision

```suggestion
```

Comment (L1ghtn1ng) [reply dns/dnscrypt-proxy/Makefile:3]: Do I remove the whole line or just leave it line as PLUGIN_REVISION=

**Comment by L1ghtn1ng** [comment]
Have committed  the change

**Comment by mimugmail** [comment]
@L1ghtn1ng You can remove pa completely, it's not used anymore. We just commet it for compatibility, but there is also no optionvalue in dropdown.

**Comment by L1ghtn1ng** [comment]
Have updated the code with required changes

**Comment by mimugmail** [comment]
Thx, looks better now .. but .. did you also test URLhaus feed? 
This feed delivers URI's in addition to domains, the filters wont match here (I did not test yet, but it looks odd).

**Comment by L1ghtn1ng** [comment]
I tested the sed commands to make sure the resulting final output text file looked tidy etc but did not actually try it as I do not know how to setup a dev environment for it to test the feed but hopefully all is in order

**Comment by mimugmail** [comment]
Sorry, it doesn't work.
It looks like this:

```
httP://185.29.10.14/welcomes.exe
httPs://tonymarislogistics.com/oo.exe
http://0-day.us/img/1.vbs
http://0-day.us/img/10.vbs
http://0-day.us/img/11.vbs
http://0-day.us/img/13.vbs
http://0-day.us/img/14.vbs
http://0-day.us/img/15.vbs
http://0-day.us/img/16.vbs
http://0-day.us/img/17.vbs
http://0-day.us/img/2.vbs
http://0-day.us/img/20.vbs
http://0-day.us/img/25.vbs
http://0-day.us/img/3.vbs
http://0-day.us/img/30.vbs
http://0-day.us/img/4.vbs
http://0-day.us/img/5.vbs
http://0-day.us/img/6.vbs
http://0-day.us/img/7.vbs
http://0-day.us/img/8.vbs
http://0-day.us/img/9.vbs
http://0-day.us/img/doc/1.doc
http://0-day.us/img/doc/10.doc
http://0-day.us/img/doc/11.doc
```

**Comment by L1ghtn1ng** [comment]
What do you suggest? Should I just remove this feed?

**Comment by mimugmail** [comment]
Just remove it. I asked abuse.ch when introducing this feed to also offer plain domain without URI, but they said this would produce too many FPs

**Comment by L1ghtn1ng** [comment]
Ah okay will do
________________________________

**Comment by L1ghtn1ng** [comment]
Done

File: dns/dnscrypt-proxy/src/opnsense/scripts/OPNsense/Dnscryptproxy/dnsbl.sh [hunk 2]
```diff
@@ -252,44 +196,18 @@ for CAT in $(echo ${DNSBL} | tr ',' ' '); do
...
-		;;
-	hpp)
-		hphosts-psh
-		;;
-	hup)
-		hphosts-pup
-		;;
-	ht)
-		hbbtv
 		;;
```
Comment (mimugmail) [inline dns/dnscrypt-proxy/src/opnsense/scripts/OPNsense/Dnscryptproxy/dnsbl.sh:205]: Remove this one too, please

**Comment by mimugmail** [comment]
I mean this one:

https://github.com/opnsense/plugins/blob/af281b9244db7d7de58a4931cd23585117f965cd/dns/dnscrypt-proxy/src/opnsense/scripts/OPNsense/Dnscryptproxy/dnsbl.sh#L198

Just go to your firewall via CLI:

```
pkg install git
git clone https://github.com/L1ghtn1ng/plugins/
cd plugins 
git checkout dnscrypt-proxy-patch
cd dns/dnscrypt-proxy
make upgrade
```

Then you can check all your code. Just execute the script returns:

```
root@OPNsense:~/test/town/plugins/dns/dnscrypt-proxy # /usr/local/opnsense/scripts/OPNsense/Dnscryptproxy/dnsbl.sh
/usr/local/opnsense/scripts/OPNsense/Dnscryptproxy/dnsbl.sh: 199: Syntax error: word unexpected (expecting ")")

```

**Comment by L1ghtn1ng** [comment]
Ah okay thanks and pushed the fix

**Comment by L1ghtn1ng** [comment]
@mimugmail can you merge this or @fichtner and or @AdSchellevis please?

**Comment by AdSchellevis** [comment]
@L1ghtn1ng sure, thanks!

**Pull request merged**